Cargo.lock

```diff
@@ -620,15 +629,34 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1b1b9d958c2b1368a663f05538fc1b5975adce1e19f435acceae987aceeeb369"
 dependencies = [
  "lazy_static",
- "nom",
- "rust-ini",
+ "nom 5.1.2",
+ "rust-ini 0.13.0",
  "serde 1.0.145",
  "serde-hjson",
  "serde_json",
  "toml",
  "yaml-rust",
 ]
 
+[[package]]
+name = "config"
+version = "0.13.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "11f1667b8320afa80d69d8bbe40830df2c8a06003d86f73d8e003b2c48df416d"
+dependencies = [
+ "async-trait",
+ "json5",
```

Comment:
Can we disable the json5 dependency? it brings in an absolute hell of cryptographic crates

Reply:
I agree